untangle-vue-ui/source/src/components/setup/Autoupgrades.vue

```diff
@@ -1,57 +1,60 @@
 <template>
-  <v-card width="1100" height="500" class="mx-auto mt-3" flat>
-    <SetupLayout />
-    <div class="auto-upgrades">
-      <h1 class="font-weight-light faint-color text-h4">{{ title }}</h1>
-      <v-container class="text-center">
-        <v-row>
-          <v-col cols="auto">
-            <v-checkbox
-              id="autoUpgrade"
-              v-model="systemSettings.autoUpgrade"
-              :label="$t('Automatically Install Upgrades')"
-              hide-details
-              class="bold-label"
-            />
-            <v-row>
-              <v-col cols="12">
-                <p class="paragraph">{{ $t('Automatically install new versions of the software when available.') }}</p>
-                <p class="paragraph">{{ $t('This is the recommended choice for most sites.') }}</p>
-              </v-col>
-            </v-row>
-          </v-col>
-        </v-row>
+  <v-container>
+    <v-card width="900" class="mx-auto mt-3 pa-3" flat>
+      <SetupLayout />
+      <div
+        class="pa-3 mt-4 mx-auto grey lighten-4 border rounded d-flex flex-column"
+        style="width: 100%; min-height: 500px; border: 1px solid #e0e0e0 !important"
+      >
+        <h1 class="font-weight-light faint-color text-h4">{{ title }}</h1>
+        <v-container class="flex-grow-1">
+          <v-row>
+            <v-col cols="auto">
+              <v-checkbox id="autoUpgrade" v-model="systemSettings.autoUpgrade" hide-details>
+                <template #label>
+                  <span class="font-weight-bold mt-2">{{ $t('Automatically Install Upgrades') }}</span>
```

Comment:
I don't think we have any translations yet for those strings. + if we would have they would be a key like `automatically_install_upgrades`, not the actual string.
So for now that does nothing, it does not hurt either (yet)